autoload/go/rename.vim

```diff
@@ -72,6 +72,8 @@ function! go#rename#Rename(bang, ...)
   " we need a way to get the list of changes from gorename upon an success
   " change.
   silent execute ":e"
+
+  call go#guru#SameIdRenamed()
```

Comment:
I think we should do it different. There should be a function called `go#guru#HasSameId()` which returns a true if it's been highlighted. And then you call it like this:

```
if go#guru#HasSameId()
  " Call sameids again to redraw the highlighting  
  call go#guru#SameIds(-1)
endif
```

Does this makes sense? I didn't like the `SameIdRenamed()` function so I would be happy if you can remove it.